docs/developer-docs/latest/developer-resources/data-management.md

```diff
@@ -146,15 +140,77 @@ npm strapi export --no-compress
 </code-block>
 </code-group>
 
+### Export only selected types of data
+
+The default `strapi export` command exports your content (entities and links), files (assets), project configuration, and schemas. The `--only` option allows you to export only the listed items by passing a comma-separated string. The available values are `content`, `files`, and `config`. Schemas are always exported, as schema matching is used for `strapi import`.
+
+
+#### Example: Export only entities and links
+<br/>
+
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --only content
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export --only content
+```
+
+</code-block>
+</code-group>
+
+### Exclude items from export
+
+The default `strapi export` command exports your content (entities and links), files (assets), project configuration, and schemas. The `--exclude` option allows you to exclude content, files, and the project configuration by passing these items in a comma-separated string. You can't exclude the schemas, as schema matching is used for `strapi import`.
+
+:::note 
+The `--exclude` option overrides the `--only` option. For example, if you included `--only files, content --exclude content` in your `strapi export` command only the `files` would be exported.
+:::
+
+#### Example: Export data excluding assets, entities, and links
+<br/>
+
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --exclude files, content
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export --exclude files, content
+```
+
+</code-block>
+</code-group>
+
 ## Import data using the CLI tool
 
 :::warning
-`strapi import` will delete all of the existing data prior to importing the backup file. Restored data does not include the `users` table, which means that `createdBy` and `updatedBy` are empty in a restored instance.  
+
+- `strapi import` will delete all existing data before importing the backup file.
+- The source and target schemas must match to successfully use `strapi import`.
+- Restored data does not include the `users` table, which means that `createdBy` and `updatedBy` are empty in a restored instance.  
```

Comment:
Should we specify this is admin users, so nobody is confused by the users and permissions users?